src/meshwifi/meshhttp.cpp

```diff
@@ -240,6 +241,7 @@ void initWebServer()
     ResourceNode *nodeRoot = new ResourceNode("/", "GET", &handleRoot);
     ResourceNode *nodeStaticBrowse = new ResourceNode("/static", "GET", &handleStaticBrowse);
     ResourceNode *nodeStaticPOST = new ResourceNode("/static", "POST", &handleStaticPost);
+    ResourceNode *nodeStaticDelete = new ResourceNode("/static", "DELETE", &handleStaticDelete);
```

Comment:
Request to move this to : /json/spiffs/delete/static

Reply:
It probably would be wise to add password protection for all routes that change the SPIFFS and Settings, if we plan to make the web interface accessable in LAN environements.

Reply:
Good idea!  Can you open a new issue in which to track that feature?

Reply:
Building in access control is on Kevin’s todo list. This is going to be built into the main device API and then inherited by the http stack.

Reply:
@mc-hamster endpoint rename done :-)